flink-python/pyflink/java_gateway.py

```diff
@@ -98,6 +98,7 @@ def preexec_func():
         gateway_parameters=GatewayParameters(port=gateway_port, auto_convert=True))
 
     # Import the classes used by PyFlink
+    java_import(gateway.jvm, "java.util.Properties")
```

Comment:
I'm suggesting to import java.util.Properties at the places they are used. We can follow the strategy to import the commonly used packages in advance and for the packages used seldom, we can put off to import them at the places they are used.

Reply:
Done.